tasks/utilities/submission/task_submission.wdl

```diff
@@ -74,15 +74,27 @@ task prune_table {
       # qc checks:
 
    
-    elif ("~{biosample_type}" == "Pathogen") or ("~{biosample_type}" == "pathogen"):
-      required_metadata = ["submission_id", "organism", "collected_by", "collection_date", "geo_loc_name", "host", "host_disease", "isolation_source", "lat_lon", "isolation_type"]
-      optional_metadata = ["sample_title", "bioproject_accession", "attribute_package", "strain", "isolate", "culture_collection", "genotype", "host_age", "host_description", "host_disease_outcome", "host_disease_stage", "host_health_state", "host_sex", "host_subject_id", "host_tissue_sampled", "passage_history", "pathotype", "serotype", "serovar", "specimen_voucher", "subgroup", "subtype", "description"] 
+    elif ("~{biosample_type}" == "Pathogen") or ("~{biosample_type}" == "pathogen") or ("Pathogen.cl" in "~{biosample_type}") or ("pathogen.cl" in "~{biosample_type}"):
```

Comment:
To avoid all of the double string checks, can this be simplified to `("~{biosample_type}".lower() == "pathogen")`?

Reply:
Not necessarily something we need to update now, but wanted to add a note in case this is a redundancy we can address as things grow.